### PR TITLE
task: add logging for when the gr4vyId is incorrect

### PR DESCRIPTION
**Pull request by luca-gr4vy**
**Description:** Adds a simple check for when the iframe doesn't load (or it takes too long), and logs a warning in that scenario.

**Ticket:** https://gr4vy.atlassian.net/browse/TA-1007

**Screenshot(s):**

Screenshot 2022-03-30 at 14 35 41

### CODE REVIEW

**Comment by cbetta** [comment]
I will leave @douglaseggleton to comment on wether this is a good technical solution. 

I do wonder if it makes more sense to only log the warning 1x after all retries fail?

File: packages/embed/src/index.ts [hunk 1]
```diff
@@ -130,6 +130,14 @@ export function setup(setupConfig: SetupConfig): EmbedInstance {
   // Attach elements to the DOM
   config.element.append(overlay, loader, frame)
 
+  // If the iframe failed to load or it took too much time, log a warning.
+  // This won't be displayed if the timer is cleared in frameReady.
+  const frameLoadWarn = setTimeout(() => {
+    console.warn(
+      'Loading Embed UI failed or took too long. Please check that the `gr4vyId` and `environment` values are correct.'
+    )
```
Comment (cbetta) [inline packages/embed/src/index.ts:138]: @luca-gr4vy can we log the props passed to embed here as well? Much like we did in EmbedUI? No need to decrypt the token as this isn't an auth issue.

**Review by cbetta — CHANGES_REQUESTED** [review]

**Review by douglaseggleton — APPROVED** [review]

File: packages/embed/src/index.ts (hunk 1 above)
Comment (luca-gr4vy) [reply packages/embed/src/index.ts:141]: ok, done

**Review by cbetta — APPROVED** [review]

**Comment by gr4vy-code** [comment]
:rocket: PR was released in [`v2.9.0`](https://github.com/gr4vy/gr4vy-embed/releases/tag/v2.9.0) :rocket:

**Pull request merged**